data/tekton-pipelines/okd/windows-efi-installer-pipeline.yaml

```diff
@@ -258,9 +258,8 @@ spec:
               source:
                 http:
                   url: "$(params.winImageDownloadURL)"
-              pvc:
-                accessModes:
-                  - ReadWriteOnce
+              storage:
+                volumeMode: "Filesystem"
```

Comment:
Why is the `volumeMode: "Filesystem"` required and would it work on ceph storage?

Reply:
when the volumeMode is not defined, I am encountering https://bugzilla.redhat.com/show_bug.cgi?id=1917908. I am trying to investigate how different volumeMode will behave.